packages/database/src/migrations/20200826215113-UseIndicatorsInStriveVisualisations-modifies-data.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+import { generateId, insertObject } from '../utilities';
+import {
+  indicators,
+  reportChanges,
+  overlayChanges,
+} from './migrationData/20200826215113-UseIndicatorsInStriveVisualisations-modifies-data';
```

Comment:
Importing details from one migration in another is new to me, but I guess it works!

Reply:
I wonder if it should happen in the same other file though, so that the next dev who adds an indicator can see that adding to the data sources needs to be handled

Reply:
It did puzzle me a little bit at first reading, however I'm not fussed since I feel migrations shouldn't be expected to be very readable / source of information.

Reply:
This doesn't import data from another migration: the data exist in a file under `migrationData`. It's a pattern we have used in the past to make migrations more readable by not including the actual data in the migration file.

The convention we use for those is naming them the same as the migration, which I find nice. Perhaps we could update the convention to make this more explicit though?

Reply:
Right - sorry, I misread in my rapid high level review! The way you've done it makes it very repeatable and will ensure other devs do pick up the part to add data sources :-)

I like the convention as well, but I think we could drop the `modifies-data`.

Reply:
I will do that then! 👍